### PR TITLE
Add snake to site

### PR DESCRIPTION
**Pull request by kylew1208**
Added link to a page with the snake game on the home page.

### CODE REVIEW

**Review by Heroshane1 — APPROVED** [review]
Looks good to me, at least for now. In the future, this will need mobile support and perhaps updated graphics.

File: BucStop/Views/Snake/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            margin: 0;
+        }
+
+        body {
+            background: black;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
```
Comment (chrisseals98) [inline BucStop/Views/Snake/Index.cshtml:23]: Why don't we just remove this if it isn't needed?

**Review by klucher — APPROVED** [review]
I approve

File: BucStop/Views/Snake/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        function getRandomInt(min, max) {
+          return Math.floor(Math.random() * (max - min)) + min;
+        }
+
+        // game loop
+        function loop() {
+          requestAnimationFrame(loop);
+
+          // slow game loop to 15 fps instead of 60 (60/15 = 4)
+          if (++count < 4) {
```
Comment (chrisseals98) [inline BucStop/Views/Snake/Index.cshtml:70]: This is how we can slow the game down if it is too fast, maybe a difficulty selection could be added in future? Just an idea

File: BucStop/Controllers/SnakeController.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace BucStop.Controllers
+{
+    public class SnakeController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
```
Comment (chrisseals98) [inline BucStop/Controllers/SnakeController.cs:9]: Do we need a whole controller for snake? We could just make a controller for games, and add a IActionResult method for Snake to take you to the snake page. The URL in this example would be 1.1.1.1/games/snake

**Pull request merged**